hack/patch_node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
+    echo "Killing proxy (pid=$proxy_pid)..."
+    kill "$proxy_pid"
+  fi
+  exit "$rc"
+}
+
+# creates/removes prow/sriov resources for the given $node
+function patch_node {
+  local node=$1
+  local capacity=$2
+  if [ "$capacity" -ne 0 ]; then
+    curl --header "Content-Type: application/json-patch+json" \
+      --request PATCH \
+      --data '[{"op": "add", "path": "/status/capacity/prow~1sriov", "value": "'$capacity'"}]' \
+      http://localhost:$PROXY_PORT/api/v1/nodes/"$node"/status
+  else
+    curl --header "Content-Type: application/json-patch+json" \
+      --request PATCH \
+      --data '[{"op": "remove", "path": "/status/capacity/prow~1sriov"}]' \
```

Comment:
maybe worth mentioning, for future vistors in this code, that the `~1` actually means `/` 
accordung to https://kubernetes.io/docs/tasks/administer-cluster/extended-resource-node/.
Maybe as an inline comment.

Reply:
I dont think we want to keep comments ?
unless you insist, there is a comment already that say that the resource name will be prow/sriov
so it would be understandable that ~1 is the /

Reply:
> I dont think we want to keep comments ?
> unless you insist, 

> there is a comment already that say that the resource name will be prow/sriov

Maybe add it a short explanation there?

> so it would be understandable that ~1 is the /

I am not sure it will be understandable (also k8s people thought it could be confusing :) ) but I won't insist.